### PR TITLE
Implementation of DamSqlDBWithMultiShard project in Public folder

### PR DESCRIPTION
**Pull request by supragyan**
1. Created DamSqlDbWithMultiShard.cs class to support Dam functionality using MultiShardQuery.
2. Created MultiShardConfiguration.cs class to implement Shard properties and methods

### CODE REVIEW

**Comment by supragyan** [comment]
@daisukenishino 

We created new feature branch as damultishard-implmt which contains only DamSqlDbWithMultiShard changes.
In previous feature branch some conflicts has occured so we created this one.

Please review and merge

File: root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```
Comment (daisukenishino2) [inline root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard/Properties/AssemblyInfo.cs]: @supragyan 

Please refer to the following.

see : https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/root_VS2012/programs/C%23/Frameworks/Infrastructure/Public/Db/DamPstGrS/Properties/AssemblyInfo.cs

Comment (supragyan) [inline root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard/Properties/AssemblyInfo.cs]: @daisukenishino 

\>Please refer to the following.
Ok.

In [History comment](https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/root_VS2012/programs/C%23/Frameworks/Infrastructure/Public/Db/DamPstGrS/Properties/AssemblyInfo.cs#L30),
Do we need to add our author details or not required?
Please confirm.

In [AssemblyInfo.cs](https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/root_VS2012/programs/C%23/Frameworks/Infrastructure/Public/Db/DamPstGrS/Properties/AssemblyInfo.cs#L45),
Do we need to update Copyright version as 
[assembly: AssemblyCopyright("Copyright (C) 2007,2014 Hitachi Solutions,Ltd.")]
  ▼ ▼ ▼
[assembly: AssemblyCopyright("Copyright (C) 2007,2016 Hitachi Solutions,Ltd.")]

Please confirm.

**Comment by daisukenishino2** [comment]
@supragyan 

This pull request will merge after next week's review.

At the moment, I couldn't found problem in this pullrequest, 
Please perform the development of the sample parallelly to this task.

**Comment by supragyan** [comment]
@daisukenishino 
\>At the moment, I couldn't found problem in this pullrequest, 
As per [148 comment](https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/148/files/ae1f465b5bae316be397b1b525d97cd002b73e6f#r61889572),
We resolved and did code commit again.
So we asked to close the pull request.
But you can merge this pull request also.

\>Please perform the development of the sample parallelly to this task.
Yes we already modified in sample parallelly and also implemented code for Data Dependent Routing.
So we will deliver this sample with both changes.

**Pull request closed without merging**